x-pack/plugins/cases/kibana.json

```diff
@@ -23,6 +23,7 @@
      "features",
      "kibanaReact",
      "kibanaUtils",
+     "ruleRegistry",
```

Comment:
We depend on the rule registry to interact with the alerts now.

Reply:
`ruleRegistry` should be optional. Cases should still work without rule registry enabled.